tests/performance/README.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Performance Testing for StackRox"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This document is an idea of executable documentation. It can be used with VS Code and `Jupyter` plugin. It's pretty handy because showing `Outline` allows faster document navigation."
```

Comment:
I'd suggest to also add a brief `Readme.md` to `tests/performance` which will explain the moving parts (jupiter, etc), what to install, and how to start, i.e., "docs are in readme.ipynb, install jupyter to render and maybe a plugin for your code editor.